app/scripts/controllers/metametrics.js

```diff
@@ -658,6 +688,15 @@ export default class MetaMetricsController {
       referrer,
       environmentType = ENVIRONMENT_TYPE_BACKGROUND,
     } = rawPayload;
+
+    ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
+    const mmiProps = {};
+
+    if (this.extension?.runtime?.id) {
+      mmiProps.extensionId = this.extension.runtime.id;
```

Comment:
Isn't this a static value? Seems odd to track that as a metric property

Reply:
Makes sense!

In that case, maybe we could establish a way to inject static build-specific values like this into the constructor, rather than embedding them directly inside the controller using a code fence. That would avoid the need for fences within the controller.

Not critical though, we can explore that in a later PR.